desktop/apps/articles/templates/section.jade

```diff
@@ -23,3 +23,6 @@ block body
         hr
         +share(section.get('title') + ' on Artsy')
   include full_feed
+
+  if veniceBanner != undefined
```

Comment:
Recently was told it is faster to do it this way, do you think there is a difference?